iOS/Flows/Conversation/Input/InputHandlerCoordinator.swift

```diff
@@ -103,6 +103,7 @@ class InputHandlerCoordinator<Result>: PresentableCoordinator<Result>,
             Task.onMainActorAsync {
                 guard let kind = try? await AttachmentsManager.shared.getMessageKind(for: first, body: text) else { return }
                 self.inputHandlerViewController.swipeableVC.currentMessageKind = kind
+                self.inputHandlerViewController.swipeableVC.inputState = .collapsed
```

Comment:
This feels right to me, would be weird to go back into an expanded state.

Reply:
Sure, let's try it out.